### PR TITLE
Added new header to capture customattributes

### PR DESCRIPTION
**Pull request by mtalreja16**

### CODE REVIEW

**Review by theganyo — CHANGES_REQUESTED** [review]

File: server/metadata_context.go [hunk 1]
```diff
@@ -46,15 +47,16 @@ func encodeExtAuthzMetadata(api string, ac *auth.Context, authorized bool) *stru
...
+		headerAccessToken:      stringValueFrom(ac.AccessToken),
+		headerAPI:              stringValueFrom(api),
+		headerAPIProducts:      stringValueFrom(strings.Join(ac.APIProducts, ",")),
+		headerApplication:      stringValueFrom(ac.Application),
+		headerClientID:         stringValueFrom(ac.ClientID),
+		headerDeveloperEmail:   stringValueFrom(ac.DeveloperEmail),
+		headerEnvironment:      stringValueFrom(ac.Environment()),
+		headerOrganization:     stringValueFrom(ac.Organization()),
+		headerScope:            stringValueFrom(strings.Join(ac.Scopes, " ")),
+		headerCustomAttributes: stringValueFrom(ac.CustomAttributes),
```
Comment (theganyo) [inline server/metadata_context.go:59]: This header should be only be added if `ac.CustomAttributes` contains a value.

**Review by theganyo — CHANGES_REQUESTED** [review]

File: server/metadata_context.go (hunk 1 above)
Comment (mtalreja16) [reply server/metadata_context.go:59]: ahhh make sense...

File: server/metadata_context.go [hunk 2]
```diff
@@ -56,6 +57,11 @@ func encodeExtAuthzMetadata(api string, ac *auth.Context, authorized bool) *stru
 		headerOrganization:   stringValueFrom(ac.Organization()),
 		headerScope:          stringValueFrom(strings.Join(ac.Scopes, " ")),
 	}
+
+	if ac.CustomAttributes != "" {
```
Comment (theganyo) [inline server/metadata_context.go:61]: Just one more quick thing: Could you please add a test to verify that this header isn't added when there is no value?

**Review by theganyo — APPROVED** [review]

**Comment by theganyo** [comment]
Thank you!

**Pull request merged**